lib/logmsg/tests/test_tags.c

```diff
@@ -159,55 +141,66 @@ test_msg_tags(void)
 
 }
 
-void
-test_filters(gboolean not)
+ParameterizedTestParameters(tags, test_filters)
+{
+  const gboolean not[] =
+  {
+    TRUE,
+    FALSE
+  };
+  return cr_make_param_array(const gboolean, not, sizeof(not) / sizeof(const gboolean));
+}
+
+ParameterizedTest(const gboolean *not, tags, test_filters)
 {
   LogMessage *msg = log_msg_new_empty();
   FilterExprNode *f = filter_tags_new(NULL);
   guint i;
   GList *l = NULL;
+  gchar *name;
 
-  test_msg("=== filter tests %s===\n", not ? "not " : "");
+  cr_log_info("=== filter tests %s===\n", not[0] ? "not " : "");
 
   for (i = 1; i < FILTER_TAGS; i += 3)
     l = g_list_prepend(l, get_tag_by_id(i));
 
   filter_tags_add(f, l);
-  f->comp = not;
+  f->comp = not[0];
```

Comment:
`not` should not be indexed in the test. (`*not`)

Reply:
Ok. I will change it. Thanks.